src/lib/encoder/Encoder.h

```diff
@@ -15,8 +15,8 @@ class Encoder : public SimpleEncoder
 	using SimpleEncoder::SimpleEncoder;
 
 	unsigned encode(const std::string& filename, std::string output_prefix);
-	unsigned encode_fountain(const std::string& filename, std::string output_prefix, int compression_level=6, double redundancy=1.2, int canvas_size=0);
-	unsigned encode_fountain(const std::string& filename, const std::function<bool(const cv::Mat&, unsigned)>& on_frame, int compression_level=6, double redundancy=4.0, int canvas_size=0);
+	unsigned encode_fountain(const std::string& filename, std::string output_prefix, int compression_level=16, double redundancy=1.2, int canvas_size=0);
```

Comment:
Updated default compression (what cimbar_js will use) to 16. Arguably we could go further, but this seems to be in the right place on the performance <-> size curve.